file-formats/zif_aff_types_v1.intf.abap

```diff
@@ -98,8 +96,6 @@ INTERFACE zif_aff_types_v1 PUBLIC.
     "! <p class="shorttext">Header for Non-Source Code Objects (no key user)</p>
     "! The header for an ABAP main object (without source code) with a description of 60 characters (no key user)
     BEGIN OF ty_header_60_cloud,
```

Comment:
Shall we remove the `ty_header_60`? We could add it to my PR in which I need it. Otherwise all PRs will see this abaplint error which might lead to a broken window effect

Reply:
Options:
A: delete it in main, its not used, ie. its not needed. Introduce it with the PR that needs the type
B: Change rule unused_types to be a warning, note that this warning will still need to be fixed, and will also propagate to all other open PRs

IMHO, just delete it

Reply:
My suggestion was A (delete it and introduce it in mine, where we need it)